src/app/tabs/test-tab.js

```diff
@@ -153,8 +153,15 @@ module.exports = class TestTab extends ViewPlugin {
     return new Promise((resolve, reject) => {
       let runningTest = {}
       runningTest[path] = { content }
-      let currentCompilerUrl = this.baseurl + '/' + this.compileTab.getCurrentVersion()
-      remixTests.runTestSources(runningTest, currentCompilerUrl, canUseWorker(this.compileTab.getCurrentVersion()), () => {}, () => {}, (error, result) => {
+      const {currentVersion, evmVersion, optimize} = this.compileTab.getCurrentCompilerConfig()
+      const currentCompilerUrl = this.baseurl + '/' + currentVersion
+      const compilerConfig = {
+        currentCompilerUrl,
+        evmVersion,
+        optimize,
+        usingWorker: canUseWorker(currentVersion)
+      }
+      remixTests.runTestSources(runningTest, compilerConfig, () => {}, () => {}, (error, result) => {
```

Comment:
this is exacly for this kind of case that I would recommend an object argument instead of a list of ordered arguments (see comment [here](https://github.com/ethereum/remix/pull/1410#discussion_r371176679)).